### PR TITLE
Feat/#378 closing app post app

### PR DESCRIPTION
**Pull request by ciaranschutte**
- use `closedBy` date instead of `expiresAt` date for closed applications
- use red styling to signify closed state
- ensure collaborator functions are disabled is application is closed

### CODE REVIEW

File: components/pages/Applications/ApplicationForm/Forms/Collaborators/index.tsx [hunk 1]
```diff
@@ -83,7 +83,11 @@ const Collaborators = ({
   const containerRef = createRef<HTMLDivElement>();
   const { fetchWithAuth } = useAuthContext();
   const theme = useTheme();
-  const disableActions = [ApplicationState.REVIEW, ApplicationState.REJECTED].includes(applicationState);
+  const disableActions = [
+    ApplicationState.REVIEW,
+    ApplicationState.REJECTED,
+    ApplicationState.CLOSED,
+  ].includes(applicationState);
```
Comment (ciaranschutte) [inline components/pages/Applications/ApplicationForm/Forms/Collaborators/index.tsx:90]: lots of prettier changes but it's just this one adding `closed` state conditional to disabling actions

File: components/pages/Applications/ApplicationForm/Header/Actions.tsx [hunk 2]
```diff
@@ -151,6 +153,18 @@ const HeaderActions = ({
               </div>
             )}
           >
+            {state === ApplicationState.APPROVED && (
```
Comment (ciaranschutte) [inline components/pages/Applications/ApplicationForm/Header/Actions.tsx:156]: Banner for approved deletion modal

File: components/pages/Applications/ApplicationForm/Header/Details.tsx [hunk 3]
```diff
@@ -24,20 +24,39 @@ import Typography from '@icgc-argo/uikit/Typography';
 import { APPLICATIONS_PATH } from 'global/constants';
 import { useTheme } from '@icgc-argo/uikit/ThemeProvider';
 
+const Expiry = ({ expiry, isExpired }: { expiry: string; isExpired: boolean }) => {
```
Comment (ciaranschutte) [inline components/pages/Applications/ApplicationForm/Header/Details.tsx:27]: conditional text / styling

File: components/pages/Applications/ManageApplications/utils.tsx [hunk 4]
```diff
@@ -97,10 +99,22 @@ export const tableColumns: TableColumnConfig<ApplicationRecord> & {
...
     id: ApplicationsField.expiresAtUtc,
     accessor: 'accessExpiry',
-    Cell: ({ original }: { original: ApplicationRecord }) =>
-      original.accessExpiry
-        ? formatDate(new Date(original.accessExpiry), DATE_RANGE_DISPLAY_FORMAT)
-        : null,
+    Cell: ({ original }: { original: ApplicationRecord }) => {
+      const theme = useTheme();
+      return (
+        <div
```
Comment (ciaranschutte) [inline components/pages/Applications/ManageApplications/utils.tsx:105]: conditional text / styling

**Review by samrichca — APPROVED** [review]

**Pull request merged**